tendermint/src/rpc/endpoint/abci_info.rs

```diff
@@ -27,41 +27,58 @@ impl rpc::Response for Response {}
 
 /// ABCI information
 #[derive(Clone, Debug, Deserialize, Serialize)]
+#[serde(default)]
 pub struct AbciInfo {
     /// Name of the application
     pub data: String,
 
     /// Version
+    #[serde(skip_serializing_if = "Option::is_none")]
```

Comment:
Interesting. So this usually would be serialized to `null` but now we skip this completely in the resulting JSON (I didn't check if the go code also omits this but it probably works anyways).

Reply:
Yup, exactly.